CSharp/DevVmPowershell/Helpers.Tests.Integration/Tests/SmokeTestHelperTests.cs

```diff
@@ -78,12 +79,25 @@ public async Task WaitForSmokeTestToCompleteAsyncTest()
 					.CreateSingleWorkspaceAsync(Constants.Workspace.DEFAULT_WORKSPACE_TEMPLATE_NAME, workspaceName, true).Result;
 
 				//Install Smoke Test Application in Workspace
-				bool installationResult = await ApplicationInstallHelper.InstallApplicationFromApplicationLibraryAsync(workspaceName, Constants.SmokeTest.Guids.ApplicationGuid);
+				string binFolderPath = Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().Location);
+				if (string.IsNullOrWhiteSpace(binFolderPath))
+				{
+					throw new Exception($"{nameof(binFolderPath)} is invalid.");
+				}
+				string rapLocation = Path.Combine(binFolderPath, TestConstants.SMOKE_TEST_APP_FILE_PATH);
+				bool installationResult = await ApplicationInstallHelper.InstallApplicationFromRapFileAsync(workspaceName, rapLocation);
 				if (!installationResult)
 				{
 					throw new Exception("Smoke Test Application failed to Install");
 				}
 
+				//Install Processing Application in Workspace
+				bool processingInstallationResult = await ApplicationInstallHelper.InstallApplicationFromApplicationLibraryAsync(workspaceName, Constants.Processing.Guid);
+				if (!processingInstallationResult)
+				{
+					throw new Exception("Processing Application failed to Install");

```

Comment:
Error message could be more descriptive like -- "Processing Application failed to Install in a workspace (<insert_workspace_id_here">)" which helps in troubleshooting.

Reply:
Done